uniffi_bindgen/src/interface/types/mod.rs

```diff
@@ -200,6 +201,10 @@ impl From<&Type> for FfiType {
                 kind: ExternalKind::Interface,
                 ..
             } => FfiType::RustArcPtr(name.clone()),
+            Type::External {
+                kind: ExternalKind::CallbackInterface,
+                ..
+            } => FfiType::UInt64,
```

Comment:
not super sure about this - I saw that the regular callback interface is a UInt64 so I just did the same thing here